app/src/main/java/com/github/swent/echo/compose/map/Map.kt

```diff
@@ -22,11 +34,45 @@ fun <T : View> EchoAndroidView(
     events: List<Event>,
     callback: (Event) -> Unit = {}
 ) {
-    AndroidView(
-        modifier = modifier.testTag("mapAndroidView"),
-        factory = factory,
-        update = { update(it, events, callback) }
+    val permissions = arrayOf(
+        Manifest.permission.ACCESS_COARSE_LOCATION,
+        Manifest.permission.ACCESS_FINE_LOCATION
     )
+    fun permissionsDenied(context: Context) = permissions.any {
```

Comment:
why is this function declared inside the EchoAndroidView function ?

Reply:
Good catch! I've moved it out.